content/docs/user-guide/assets/asset-processor/faster-scanning.md

```diff
@@ -6,7 +6,24 @@ weight: 400
 toc: true
 ---
 
-**Faster Scanning Mode** speeds up **Asset Processor’s** startup scan by using timestamps to track source asset changes rather than file hashes. Faster Scanning Mode also skips checks for cache changes that occurred while Asset Processor was not running. This can save time when processing many assets in a project. You can switch scan modes at any time without restarting the Asset Processor, including during an asset scan. Asset Processor saves your preference between sessions.
+**Faster Scanning Mode** speeds up **Asset Processor’s** startup scan by using timestamps to track source asset changes before checking file hashes. This saves time when launching Asset Processor in all circumstances. You can switch scan modes at any time without restarting the Asset Processor, including during an asset scan. Asset Processor saves your preference between sessions.
+
+With Faster Scanning enabled, the process is:
+1. The Asset Processor compares the timestamp of the source asset's last modification time against the last time it processed that file. If the timestamp is identical, it skips processing that source asset.
+1. If the timestamps were different, it then creates a hash of the contents of the source asset and compares that to the hash of the source asset the last time it was processed. If they match, it skips processing the source asset.
+    * There are a few reasons why time stamps may not match but the hash of the content matches:
+        * Some source control configurations may result in the same source asset having a different timestamp.
+        * Some archival formats, like zip, will truncate timestamps, so a source asset that is placed into a zip file and removed will have a truncated timestamp.
+        * Sometimes content creators may re-save a file without making a change.
+1. If the timestamp and hash are different, then this starts following the standard asset processing process, calling create jobs on the source asset for all relevant builders.
```

Comment:
This is much better, the only thing I would add is that there's a final requirement that the builder's fingerprint and the file fingerprint haven't changed.  The file fingerprint is really just checking the dependencies haven't changed, so you can just say its checking the source/job dependencies haven't changed.